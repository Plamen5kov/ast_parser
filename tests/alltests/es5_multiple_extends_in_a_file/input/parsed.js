(function () {
    var MyButton = android.widget.Button1111111111.extend("my.custom.ClassFromOrigina1111111", {
        onClick111111: function () {
            console.log('click happened');
        },
        onClick111111111: function () {
            console.log('click happened');
        }
    });
})();
var MyButton = android.widget.Button22222222.extend("my.custom.ClassFromOrigina2222222222", {
	onClick222222: function () {
		console.log('click happened');
	},
	onClick222222222: function () {
		console.log('click happened');
	}
});
var MyButton = android.widget.Button333333.extend("my.custom.ClassFromOrigina3333333333", {
	onClick33333: function () {
		console.log('click happened');
	},
	onClick333333333: function () {
		console.log('click happened');
	}
});
(function () {
    var MyButton = android.widget.Button44444444.extend("my.custom.ClassFromOrigina44444", {
        onClick4444: function () {
            console.log('click happened');
        },
        onClick4444444: function () {
            console.log('click happened');
        }
    });
})();
