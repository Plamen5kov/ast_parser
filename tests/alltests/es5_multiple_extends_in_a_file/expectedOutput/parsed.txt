Java File: my.custom.ClassFromOrigina1111111 - Extend Class: android.widget.Button1111111111 - Overridden Methods: onClick111111,onClick111111111
Java File: my.custom.ClassFromOrigina2222222222 - Extend Class: android.widget.Button22222222 - Overridden Methods: onClick222222,onClick222222222
Java File: my.custom.ClassFromOrigina3333333333 - Extend Class: android.widget.Button333333 - Overridden Methods: onClick33333,onClick333333333
Java File: my.custom.ClassFromOrigina44444 - Extend Class: android.widget.Button44444444 - Overridden Methods: onClick4444,onClick4444444